### DISCUSSION TITLE
Request for Docker Image

### DISCUSSION THREAD

**Discussion by preethaml7** [ananthakumaran/paisa#35] [Ideas]
Hello! I've been following the progress of this project and I'm interested in self-hosting it. I noticed that there isn't currently a Docker image available for easy installation. Are there any plans to create a Docker image for this project in the future? It would be incredibly helpful for users like me who prefer Docker over traditional package installations. Thank you for your hard work on this project!

**Comment by avinashkanaujiya** [comment]
+1

**Reply by ananthakumaran** [reply]
already available https://paisa.fyi/getting-started/installation/#docker
